.goreleaser.yml

```diff
@@ -49,3 +50,83 @@ brews:
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.licenses=MIT License
+  - image_templates: [ "ministryofjustice/{{ .ProjectName }}:{{ .Tag}}-arm" ]
+    goarch: arm
+    dockerfile: '{{ .Env.DOCKERFILE }}'
+    use: buildx
+    build_flag_templates:
+      - --build-arg=COMMAND={{ .ProjectName }}
+      - --platform=linux/arm
+      - --label=org.opencontainers.image.title={{ .ProjectName }}
+      - --label=org.opencontainers.image.description={{ .ProjectName }}
+      - --label=org.opencontainers.image.url=https://github.com/ministryofjustice/{{ .ProjectName }}
+      - --label=org.opencontainers.image.source=https://github.com/ministryofjustice/{{ .ProjectName }}
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.licenses=MIT License
+  - image_templates: [ "ministryofjustice/{{ .ProjectName }}:{{ .Tag}}-386" ]
+    goarch: "386"
+    dockerfile: '{{ .Env.DOCKERFILE }}'
+    use: buildx
+    build_flag_templates:
+      - --build-arg=COMMAND={{ .ProjectName }}
+      - --platform=linux/386
+      - --label=org.opencontainers.image.title={{ .ProjectName }}
+      - --label=org.opencontainers.image.description={{ .ProjectName }}
+      - --label=org.opencontainers.image.url=https://github.com/ministryofjustice/{{ .ProjectName }}
+      - --label=org.opencontainers.image.source=https://github.com/ministryofjustice/{{ .ProjectName }}
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.licenses=MIT License
+
+docker_manifests: 
+  - name_template: "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}"
+    image_templates:
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-amd64"
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-arm64"
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-arm"
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-386"
+  - name_template: "ministryofjustice/{{ .ProjectName }}:latest"
+    image_templates: 
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-amd64"
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-arm64"
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-arm"
+      - "ministryofjustice/{{ .ProjectName }}:{{ .Tag }}-386"
+
+announce:
+  slack:
+    enabled: true
+    message_template: '{{ .ProjectName }} {{ .Tag }} is now live: {{ .ReleaseURL }}'
```

Comment:
I like this idea. I'm not sure it truly states what we're releasing. `is now live:` is misleading.

Reply:
does this make more sense
message_template: '{{ .ProjectName }} {{ .Tag }}: The latest release has been created successfully: {{ .ReleaseURL }}'

Reply:
updated the output text